### PR TITLE
Downgrade API container python to 3.10 from 3.11 to fix a node-gyp build error

### PR DESCRIPTION
**Pull request by LukeChannings**
I get the following error when building the API container on ARM:

```
395.0 Traceback (most recent call last):
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/gyp_main.py", line 51, in <module>
395.0     sys.exit(gyp.script_main())
395.0              ^^^^^^^^^^^^^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 670, in script_main
395.0     return main(sys.argv[1:])
395.0            ^^^^^^^^^^^^^^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 662, in main
395.0     return gyp_main(args)
395.0            ^^^^^^^^^^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 629, in gyp_main
395.0     [generator, flat_list, targets, data] = Load(
395.0                                             ^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 150, in Load
395.0     result = gyp.input.Load(
395.0              ^^^^^^^^^^^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 3021, in Load
395.0     LoadTargetBuildFile(
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 411, in LoadTargetBuildFile
395.0     build_file_data = LoadOneBuildFile(
395.0                       ^^^^^^^^^^^^^^^^^
395.0   File "/app/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 239, in LoadOneBuildFile
395.0     build_file_contents = open(build_file_path, "rU").read()
395.0                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
395.0 ValueError: invalid mode: 'rU' while trying to load binding.gyp
395.0 gyp ERR! configure error
395.0 gyp ERR! stack Error: `gyp` failed with exit code: 1
395.0 gyp ERR! stack     at ChildProcess.onCpExit (/app/node_modules/@npmcli/run-script/node_modules/node-gyp/lib/configure.js:351:16)
395.0 gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
395.0 gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
395.0 gyp ERR! System Linux 6.4.16-linuxkit
395.0 gyp ERR! command "/usr/local/bin/node" "/app/node_modules/.bin/node-gyp" "rebuild"
395.0 gyp ERR! cwd /app/node_modules/microtime
395.0 gyp ERR! node -v v18.16.1
395.0 gyp ERR! node-gyp -v v7.1.2
395.0 gyp ERR! not ok
```

An investigation led me to https://github.com/nodejs/node-gyp/issues/2219, which indicates this is a bug introduced in Python 3.11. Setting the Alpine version to 3.17 effectively downgrades Python from 3.11 to 3.10 (there's only one Python version bundled in each Alpine release).

It seems like this might only affect ARM64, [since CI is currently using 3.11](https://github.com/omnivore-app/omnivore/actions/runs/6575606292/job/17863141470#step:3:90) without encountering this issue.

### CODE REVIEW

**Comment by jacksonh** [comment]
Ah funny timing, i just pushed a version that moves off of alpine for the core containers.

**Comment by LukeChannings** [comment]
Any reason we can't move away from Alpine in general? It seems that the compromise between image size and musl performance problems aren't a good trade-off, I think the Node/JS stack is less impacted than Python, but anything using node-gyp binaries could be affected - https://pythonspeed.com/articles/alpine-docker-python/.

I think the problem that this PR solves will still be present in your PR, since it's the Python version that's the culprit. I'll build your PR locally and see if I can reproduce this bug anyway. Otherwise I'll update this PR against your lerna upgrade, installing a specific Python version in debian might get a bit wordy though!

**Comment by LukeChannings** [comment]
I built locally against your Lerna PR and this problem goes away!

I'll close this and wait for your PR to be merged.
